build.gradle

```diff
@@ -2,7 +2,8 @@ apply plugin: 'com.github.kt3k.coveralls'
 apply plugin: 'jacoco'
 
 buildscript {
-  apply from: "${rootDir}/gradle/dependencies.gradle"
+    ext.kotlin_version = '1.3.50'
```

Comment:
This is not necessary, we already have `kotlinStdLib` dependency defined in [`dependencies.gradle`](https://github.com/mapbox/mapbox-navigation-android/blob/master/gradle/dependencies.gradle#L38)